clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1275,7 +1276,8 @@ RequestFuture<Void> sendOffsetCommitRequest(final Map<TopicPartition, OffsetAndM
         Node coordinator = checkAndGetCoordinator();
         if (coordinator == null)
             return RequestFuture.coordinatorNotAvailable();
-
+        Map<String, Uuid> topicIds = metadata.topicIds();
```

Comment:
Let's focus on the CommitRequestManager to start with. We can keep using topic names in this one for now.